### PR TITLE
specify downstreamIgnoreList in CI

### PR DESCRIPTION
**Pull request by dhmlau**
### Description
Related to https://github.com/strongloop/loopback-connector-mongodb/issues/509

- Specify the `downstreamIgnoreList` in `package.json` 
  - see reference https://github.com/strongloop/loopback-datasource-juggler/blob/master/package.json#L64-L71
- Update CODEOWNERS to reflect current status

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "4.2.0",
   "description": "The official MongoDB connector for the LoopBack framework.",
   "engines": {
-    "node": ">=6"
+    "node": ">=8"
```
Comment (bajtos) [inline package.json]: Bumping up the minimum supported Node.js version is often considered as a breaking change requiring semver-major release. What is our current position?

File: CODEOWNERS [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
 # Each line is a file pattern followed by one or more owners,
-# the last matching pattern has the most precendence.
+# the last matching pattern has the most precedence.
 
 # Current maintainers
-* @jannyHou @loay @b-admike @virkt25 @dhmlau @shimks @nitro404
+* @jannyHou @b-admike @dhmlau @nitro404 @emonddr
+
+# Alumni maintainers
+# @loay @virkt25 @shimks
```
Comment (bajtos) [inline CODEOWNERS]: Please don't mix unrelated changes in one commit. I don't see how are updates to CODEOWNERS related to downstreamIgnoreList?

File: package.json [hunk 3]
```diff
@@ -45,5 +45,12 @@
     "semver": "^5.5.1",
     "should": "^13.2.1",
     "sinon": "^6.1.3"
+  },
+  "ci": {
+    "downstreamIgnoreList": [
+      "bluemix-metering",
+      "bluemix-service-broker",
+      "plan-manager"
```
Comment (bajtos) [inline package.json]: 👍

File: CODEOWNERS (hunk 2 above)
Comment (dhmlau) [reply CODEOWNERS]: Sorry about that. Will make it a separate commit next time.
